game.js

```diff
@@ -1,51 +1,84 @@
-const rls = require('readline-sync')
+const rls = require("readline-sync");
 
 /**
  * Starts the game by prompting the user if they want to play
  * Calls either gameLoop() or quitGame()
- * 
+ *
  * @returns {undefined}
  */
 const startGame = () => {
-
-}
+  console.log("welcome");
+  let question = rls.keyInYN("Would you like to play");
+  if (question) {
+    console.log("Let's start!");
+    gameLoop();
+  } else {
+    console.log("Have a nice life!");
+    quitGame();
+  }
+};
 
 /**
  * Logs "Goodbye!"
  * Calls process.exit() to quit the game
- * 
+ *
  * @returns {undefined}
  */
 const quitGame = () => {
-
-}
+  console.log("Goodbye!");
+  process.exit();
+};
 
 /**
  * Controls the flow of the game.
  * Should prompt the user to play again once all
  * guesses have been made or correct answer guessed
- * 
+ *
  * @returns {undefined}
  */
 const gameLoop = () => {
-
-}
-
+  console.log("I have a random number in mind");
+  console.log("It's between 1 and 1000");
+  console.log("You have 10 guesses total");
+  let guessCount = 10
+  let randomNum = generateRandomNumber();
+  while (guessCount > 0){
```

Comment:
So if a user guesses more than 10 times the game automatically quits which is okay but I'd like to see a message to the user explaining that the count limit has been reached. 

It could also be really useful for the user to see the number of guesses they have left.